dspace-uclouvain/src/main/java/org/dspace/uclouvain/degreeMappers/DegreeMappersService.java

```diff
@@ -10,6 +10,7 @@
 import org.dspace.uclouvain.configurationFiles.factory.ConfigurationFileFactory;
 import org.dspace.uclouvain.configurationFiles.files.DegreeMappersConfigurationFile;
 import org.dspace.uclouvain.degreeMappers.model.DegreeMapper;
```

Comment:
Unused import `org.springframework.beans.factory.annotation.Autowired`.

Reply:
You can also remove `import org.dspace.uclouvain.configurationFiles.ConfigurationFile`.